src/Components/OrganizationData/OrganizationDataLayout.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import  Container  from '@mui/material/container'
+
+const OrganizationDataLayout = ({information}) => {
+    const {push} = useHistory()
+
+    return ( 
+
+        <Container>
+            
+        </Container>
+    );
+}
+ 
+export default OrganizationDataLayout;
```

Comment:
I think that this file do not go here, please delete it from the PR. Duplicated file

Reply:
Done!